packages/server/lib/controllers/connection.controller.ts

```diff
@@ -506,6 +507,93 @@ class ConnectionController {
         }
     }
 
+    async bulkSetMetadata(req: Request, res: Response<any, Required<RequestLocals>>, next: NextFunction) {
+        try {
+            const environment = res.locals['environment'];
+            const providerConfigKey = (req.params['provider_config_key'] as string) || (req.get('Provider-Config-Key') as string);
```

Comment:
I don't see how it could be in the params?

Reply:
I don't either 😂. Updated with https://github.com/NangoHQ/nango/pull/2145/commits/982847dcca41db7ced7a3cc83234a9adaba80526

Reply:
I think you might have modified the wrong endpoint